docs/vsebreakdown.html

```
@@ -1,47 +1,2123 @@
- Star<!DOCTYPE html>
+Star<!doctype html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)